src/Npgsql/Schema/DbColumnSchemaGenerator.cs

```diff
@@ -150,10 +150,8 @@ internal ReadOnlyCollection<NpgsqlDbColumn> GetColumnSchema()
                     populatedColumns++;
                 }
 
-                var fieldName = field.Name.StartsWith("?column?") ? null : field.Name;
-
-                column.BaseColumnName ??= fieldName;
-                column.ColumnName = fieldName;
+                column.ColumnName = field.Name.StartsWith("?column?") ? null : field.Name;
```

Comment:
Not sure we should change this to null... Shouldn't we pass through what PostgreSQL returns here (`?column?`)?

Reply:
It was a previous logic I tried not to touch.

Reply:
Also right, sorry.

Reply:
@YohDeadfall note that we just merged #3044 which changes this logic.